stressTest.yml

```diff
@@ -0,0 +1,18 @@
+config:
+  target: "http://localhost:3000"
+  phases:
+  - duration: 10
+    arrivalRate: 5
+    name: Warm up
+  - duration: 60
+    arrivalRate: 5
+    rampTo: 2500
+    name: Ramp up load
+  - duration: 120
+    arrivalRate: 2500
+    name: Torture!
```

Comment:
LOL about the name, did this work? I had problems when creating these many users to connect to Postgres.

Reply:
I guess it "worked", sort of. All the requests did eventually resolve successfully, but it was close to an hour before my server finished sending all the responses.